src/content/docs/fr/guides/images.mdx

```diff
@@ -0,0 +1,928 @@
+---
+title: Ma Page titre
+---
+import { Image } from 'astro:assets';
+import rocket from '../assets/rocket.png';
+
+# Ma Page MDX
+
+// Image locale stockée dans src/assets/
+<Image src={rocket} alt="A rocketship in space."/>
+<img src={rocket.src} alt="A rocketship in space." />
+![A rocketship in space](../assets/rocket.png)
+
+// Image stockée dans public/images/
+<Image src="/images/stars.png" alt="A starry night sky." />
+<img src="/images/stars.png" alt="A starry night sky." />
+![A starry night sky.](/images/stars.png)
+
+// Image distante sur un autre serveur
+<Image src="https://example.com/images/remote-image.png" />
+<img src="https://example.com/images/remote-image.png" />
+![Astro](https://example.com/images/remote-image.png)
+
+```
+
+## Images dans les collections de contenus
+
+Vous pouvez déclarer une image associée à une entrée d'une collection de contenus, telle que l'image de couverture d'un article de blog, dans votre page d'accueil en utilisant son chemin d'accès relatif au dossier actuel :
+
+```md title="src/content/blog/my-post.md" {3}
+---
+title: "Mon premier article de blog"
+cover: "./firstpostcover.jpeg" # will resolve to "src/content/blog/firstblogcover.jpeg"
+coverAlt: "Photographie d'un coucher de soleil derrière une chaîne de montagnes."
+---
+
+Ceci est un article de blog
+```
+
+L'aide `image` pour le schéma des collections de contenu vous permet de valider les métadonnées de l'image en utilisant Zod.
+
+```ts title="src/content/config.ts"
+import { defineCollection, z } from "astro:content";
+
+const blogCollection = defineCollection({
+	schema: ({ image }) => z.object({
+		title: z.string(),
+		cover: image().refine((img) => img.width >= 1080, {
+			message: "L'image de couverture doit avoir une largeur d'au moins 1080 pixels!",
+		}),
+		coverAlt: z.string(),
+	}),
+});
+
+export const collections = {
+	blog: blogCollection,
+};
+```
+
+L'image sera importée et transformée en métadonnées, vous permettant de la passer comme `src` à `<Image/>`, `<img>`, ou `getImage()`.
+
+
+L'exemple ci-dessous montre une page d'index de blog qui rend la photo de couverture et le titre de chaque article de blog à partir du schéma ci-dessus :
+
+```astro title="src/pages/blog.astro" {10}
+---
+import { Image } from "astro:assets";
+import { getCollection } from "astro:content";
+const allBlogPosts = await getCollection("blog");
+---
+
+{
+	allBlogPosts.map((post) => (
+		<div>
+			<Image src={post.data.cover} alt={post.data.coverAlt} />
+			<h2>
+				<a href={"/blog/" + post.slug}>{post.data.title}</a>
+			</h2>
+		</div>
+	))
+}
+```
+
+## Images dans les composants framework UI
+
+Lors de l'ajout d'images dans un composant d'interface utilisateur, utilisez la syntaxe d'image propre au composant pour rendre une image (par exemple, `<img />` en JSX, `<img>` en Svelte).
+
+Les images locales doivent **d'abord être importées** pour accéder à leurs [propriétés d'image](#local-images-in-src) telles que `src`
```

Comment:
"#local-images-in-src" il va avoir une erreur sur ce lien

Reply:
Oui ouu je sais mais j'attendais la preview